clipper_admin/clipper_admin/kubernetes/kubernetes_container_manager.py

```diff
@@ -111,35 +114,44 @@ def start_clipper(self, query_frontend_image, mgmt_frontend_image,
                     "image"] = img
                 self._k8s_beta.create_namespaced_deployment(
                     body=body, namespace='default')
+
             with _pass_conflicts():
                 body = yaml.load(
                     open(
                         os.path.join(cur_dir, '{}-service.yaml'.format(name))))
                 self._k8s_v1.create_namespaced_service(
                     body=body, namespace='default')
+
+        start_prometheus(self._k8s_v1, self._k8s_beta)
+
```

Comment:
Shouldn't you start the `frontend_exporter` here?

Reply:
Make sense. I just need to get the query_addr again.